examples/src/bin/builder_basics.ui

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would put the ui file in a ui subdirectory though as we will probably have more of them in the future for other examples